automation/publish.sh

```diff
@@ -8,39 +8,32 @@
 
 image_registry=${IMAGE_REGISTRY:-quay.io}
 image_repo=${IMAGE_REPO:-nmstate}
-branch=${PULL_BASE_REF:-$(git rev-parse --abbrev-ref HEAD)}
-docs_source_branch=master
-docs_container=${image_registry}/${image_repo}/kubernetes-nmstate-docs:$docs_source_branch
 
-# Publish kubernets-nmstate containers
 source automation/check-patch.setup.sh
-(
+cd ${TMP_PROJECT_PATH}
+
+push_handler() {
     cd ${TMP_PROJECT_PATH}
     make \
         IMAGE_REGISTRY=${image_registry}  \
         IMAGE_REPO=${image_repo} \
         push-handler
-)
-
-# We don't have a versioned docs webpage so we only update it from master and
-# not from release branches
-if [ "$branch" == $docs_source_branch ]; then
-
-    # Update kubernetes-nmstate documentation container
-    docker build docs --build-arg BRANCH=$docs_source_branch -t $docs_container
-    docker push $docs_container
-
-    # To pass user/password from automations, idea is taken from [1]
-    # [1] https://stackoverflow.com/questions/8536732/can-i-hold-git-credentials-in-environment-variables
-    git config credential.helper '!f() { sleep 1; echo "username=${GITHUB_USER}"; echo "password=${GITHUB_TOKEN}"; }; f'
+}
 
+publish_docs() {
```

Comment:
This should happen only on the doc branch, no?

Reply:
we publish from `master` into `gh-pages` branch, is the way github pages works.